backend/src/routes/api.router.ts

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import checkHealth from '../controllers/health.api.controller';
+
+const router = express.Router();
+
+router.route('/health').get(checkHealth);
+
+export default router;
```

Comment:
Well done! 
When you add a second second route later, look into defining index files for your folders, it'll make managing the exports 100x cleaner